install

```diff
@@ -751,6 +751,12 @@ else
     fi
 fi
 
+# Manual installation of pip3 components to make PyGObject work
+echo "Manual installation of pip3 components to make PyGObject work"
+localUser=$(who am i | awk '{print $1}')
+loudCmd sudo -u $localUser pip3 install pycairo
```

Comment:
This should be part of the `setup.py` file for the python app that this breaks. The dependencies should be listed with the others there

Reply:
By the looks of the errors, these should be in `python_apps/airtime_analyzer/setup.py`

Reply:
When putting it in the way you mentioned, I encounter the same error as last time.

Also, about the dependency issue, one of the PyGObject maintainers writes:
>py3cairo is the pkg-config name, not related to pypi.
Seems you are missing pycairo, but pip should install it before since it's an install dependency, so not sure how this can happen.
Did you disable isolated builds in pip?

Reply:
So then they require a user owned version of those dependencies...? That is very strange and potentially breaks things (e.g. in a container when the user is root

Reply:
No idea.